### PR TITLE
Update Alerts Management Platform PowerShell - New Alert Processing Rule Version

### PR DESCRIPTION
**Pull request by khaledaboasbe**
## Description

new feature - alert processing rule.
alert processing rule substitute action rules.

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by wyunchi-ms** [comment]
Please resolve the conflicts.

**Comment by khaledaboasbe** [comment]
I don't why there are conflicts, I deleted these files

**Comment by VeryEarly** [comment]
Deleting cmdlets is considered breaking change for Azure-PowerShell, can you keep them both?

**Comment by khaledaboasbe** [comment]
No, we know it's breaking changes and this is what our changes must be.
we are changing these commands to be Get-AzAlertProcessingRule instead of AzActionRule
we are changing the API
we already got an exception on this breaking change on swagger.
If there are any extra actions needed to make this happen please tell me

**Comment by khaledaboasbe** [comment]
I'm not sure if you started the review or not.
so could you please review my changes, we have tight timelines

File: src/AlertsManagement/AlertsManagement/OutputModels/PSDiagnosticsActionRule.cs [hunk 1]
```diff
@@ -19,9 +19,9 @@
 
 namespace Microsoft.Azure.Commands.AlertsManagement.OutputModels
 {
-    public class PSDiagnosticsActionRule : PSActionRule
```
[the remark this answers is not in the document]
Comment (khaledaboasbe) [reply src/AlertsManagement/AlertsManagement/OutputModels/PSDiagnosticsActionRule.cs:22]: Done

**Review by VeryEarly — APPROVED** [review]

**Pull request merged**